scripts/install_narrative.sh

```diff
@@ -71,29 +69,9 @@ function usage () {
     printf "If the virtualenv does not yet exist, the script will attempt\n\tto make one for you with default options.\n"
```

Comment:
Is this true anymore?

Reply:
Nope, removed that text and tweaked the rest, and some other options.